_events/sr2022/competition.md

```diff
@@ -12,6 +12,32 @@ The Student Robotics Competition marks the ultimate stage of SR2022, showcasing
 
 The competition will be held from *Saturday 23rd April* until *Sunday 24th April*. Doors open 9am and we aim to finish around 6pm.
 
+### Day 1 - 23rd April, 2022
+
+Time | Activity
+--- | ---
+9:00 | Doors Open
+9:30 | Welcome Talk
+9:40 | Tinker Time
+11:30 | League Matches
+12:30 | Lunch and Tinker Time
+13:30 | League Matches
+17:00 | End of Day
+
+### Day 2 - 24th April, 2022
+
+Time | Activity
+--- | ---
+8:45 | Doors Open
+9:30 | Tinker
+11:00 | League Matches
+12:30 | Lunch and Tinker Time
+13:30 | Competition Photo
+14:00 | Knockouts
+16:20 | Prize Ceremony
+16:45 | Kit Return
```

Comment:
The end-of-day on Sunday is just a nominal time. In practice this whenever they've left by.

Reply:
We should still advertise a time as a "Please be gone by this time"

Reply:
On Sunday evening people may have quite a way to travel. If we're advertising an End time then they'll likely assume that's a time they can actually be leaving at, so us having a bottleneck at the exit which makes that time less realistic doesn't seem fair.

Back to my original question: is 15 minutes enough time for Kit Return?

I don't know what's involved in the return, however often it involves someone looking in their kit to ensure that everything is actually present. This takes a while per team (certainly > 2 minutes) and while it can parallelise very well desk space quickly becomes the limiting factor.

To this end, many teams often start deconstructing their robots as soon as they're knocked out of the knockouts, in part so they can get away quickly at the end of the day. We typically therefore want to have kit return actually available throughout the afternoon to facilitate this (even if we don't actually advertise it).

Has this all been considered in the allowance of only 15 minutes for Kit Return?

Reply:
Can I suggest this is discussed on the issuea that decides the schedule (https://github.com/srobo/tasks/issues/743) and kit handback (https://github.com/srobo/tasks/issues/759).

Reply:
-> https://github.com/srobo/tasks/issues/759#issuecomment-1086371945

Reply:
The conclusion in a recent Competition Team meeting was that we should ship this as-is and we can amend it if needed.